### PR TITLE
unnit test for adding a new line at the end of source files

### PR DESCRIPTION
**Pull request by Pratikrocks**
Signed-off-by: Pratik Dey <pratikrocks.dey11@gmail.com>

A fix for the issue #999

### CODE REVIEW

**Comment by Pratikrocks** [comment]
@pdxjohnny I used the recursive approach to enumerate all files and the directories .

File: tests/test_styles.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                        yield x
+        if not topdown:
+            yield top, dirs, nondirs
+
+    def test_styles(self):
+        for root, dirnames, filenames in self.walk(ROOT_DIR, maxdepth=None):
+            for file in filenames:
+                with open(root + '/' +  file ,"r") as f:
+                    mime = mimetypes.guess_type(f.name)[0]
+                    if mime is not None:
```
Comment (pdxjohnny) [inline tests/test_styles.py:49]: Use `pathlib.Path.read_text()/write_text()`

**Review by pdxjohnny — CHANGES_REQUESTED** [review]

File: tests/test_styles.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import os
+from os import path
+import mimetypes
+from pathlib import Path
+import unittest
+from dffml.util.asynctestcase import IntegrationCLITestCase
+
+ROOT_DIR = os.path.dirname(os.path.abspath(os.path.dirname(__file__)))
+
+class TestML(unittest.TestCase):
```
Comment (pdxjohnny) [inline tests/test_styles.py:13]:
```suggestion
class TestML(IntegrationCLITestCase):
```

**Comment by Pratikrocks** [comment]
Hi @pdxjohnny thanks for the review ,I will be making the suggested changes very soon 
Thanks 😊

File: tests/test_styles.py (hunk 1 above)
Comment (Pratikrocks) [reply tests/test_styles.py:49]: Thanks for these suggestions 👍

**Comment by Pratikrocks** [comment]
Hi @pdxjohnny, Sorry for the delay I made a separate PR for this, I am closing this PR as there is a small typo in the branch name 
New PR is #1015 .

**Pull request closed without merging**